blocks/identity-block/features/social-sign-on/default.jsx

```diff
@@ -58,7 +40,7 @@ const SocialSignOnBlock = ({ customFields }) => {
     <section className="xpmedia-subs-social-sign-on">
       <SocialSignOn
         onError={() => { setError(phrases.t('identity-block.login-form-error')); }}
-        redirectURL={redirectURL}
+        redirectURL={loginRedirect}
```

Comment:
this seems to have been renamed. No big deal just wanted to call it out

Reply:
Yeah just a renamed variable to make it clear